rest-client-base/src/test/java/com/wultra/core/rest/client/base/DefaultRestClientTest.java

```diff
@@ -498,4 +502,26 @@ public void testDefaultHttpHeaders() throws RestClientException {
         assertEquals(headerVaue, responseEntity.getHeaders().getFirst(headerName));
     }
 
+    @Test
+    public void testConfiguration() throws RestClientException {
```

Comment:
`public` is not needed in JUnit 5. Keeping it consistent so far. I will improve it in a separate PR.